### ISSUE TITLE
Webpack 2.2.0 - Cannot assign to read only property 'exports' of object '#<Object>'

### ISSUE THREAD

**Issue by mfazekas** [renchap/webpacker-react#6]
```sh
./bin/webpack --version
2.2.0
```

when trying `webpacker-react-example` i'm receiving 
`Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`

According to chrom it seems to be coming from this line:

```js
window.WebpackerReact.addEventEventhandlers();

module.exports = window.WebpackerReact; /* Cannot assign to read only property... */



//////////////////
// WEBPACK FOOTER
// (webpack)er-react/index.js
// module id = 81
// module chunks = 0
```

See https://github.com/webpack/webpack/issues/4039

**Comment by mfazekas** [comment]
Seems to be fixed by upgrading `pakcage.json` in webpacker-react-example

**Comment by renchap** [comment]
Yes, this has been fixed in version 0.0.4 of the `webpacker-react` NPM package. Thanks for the PR 👍
